components/atom/tooltip/src/index.scss

```diff
@@ -1,76 +1,4 @@
-@import '~@s-ui/theme/lib/index';
-
-$fz-atom-tooltip: $fz-m !default;
-$lh-atom-tooltip: $lh-m !default;
-$wlc-atom-tooltip: 4 !default; // -webkit-line-clamp → lines to show
-
-$maw-atom-tooltip: 250px !default;
-$mah-atom-tooltip: 100px !default;
-$c-atom-tooltip: $c-white !default;
-$bgc-atom-tooltip: $c-black !default;
-$bdc-atom-tooltip: $bgc-atom-tooltip !default;
-$bdrs-atom-tooltip: $bdrs-l !default;
-$op-atom-tooltip: 1 !default;
-$pt-atom-tooltip: $p-m !default;
-$pr-atom-tooltip: $p-m !default;
-$pb-atom-tooltip: $p-m !default;
-$pl-atom-tooltip: $p-m !default;
-$p-atom-tooltip: $pt-atom-tooltip $pr-atom-tooltip $pb-atom-tooltip
-  $pl-atom-tooltip !default;
-$mt-atom-tooltip: $m-m !default;
-$mb-atom-tooltip: $m-m !default;
-$ml-atom-tooltip: $m-m !default;
-$mr-atom-tooltip: $m-m !default;
-$m-atom-tooltip: $mt-atom-tooltip $mr-atom-tooltip $mb-atom-tooltip
-  $ml-atom-tooltip !default;
-$ta-atom-tooltip: left !default;
-$z-atom-tooltip: 1070 !default;
-
-$c-atom-tooltip-target: inherit !default;
-$fz-atom-tooltip-target: inherit !default;
-$ml-atom-tooltip-target: 0 !default;
-
-$w-atom-tooltip-arrow: $p-m + $p-s !default;
-$h-atom-tooltip-arrow: $p-m !default;
-$c-atom-tooltip-arrow: $bgc-atom-tooltip !default;
-
-$bdw-atom-tooltip: 0 !default;
-
-$base-class: '.sui-AtomTooltip';
-$class-inner: '#{$base-class}-inner';
-$class-arrow: '#{$base-class}-arrow';
-$class-target: '#{$base-class}-target';
-
-$c-atom-tooltip-colors: (
-  'primary': (
-    $c-primary,
-    $c-white
-  ),
-  'secondary': (
-    $c-secondary,
-    $c-white
-  ),
-  'accent': (
-    $c-accent,
-    $c-white
-  ),
-  'neutral': (
-    $c-gray,
-    $c-white
-  ),
-  'success': (
-    $c-success,
-    $c-white
-  ),
-  'alert': (
-    $c-alert,
-    $c-white
-  ),
-  'error': (
-    $c-error,
-    $c-white
-  )
-) !default;
+@import './settings.scss';
```

Comment:
is it a must to include the sui-theme in the settings file? to import the theme and then the settings sounds more appropriate to me as there should be sui-theme tokens used in the file and having it imported in the settings file makes it harder to know where those tokens come from

Reply:
But you don't see any token usage of the sui-theme here. To be honest, I see more appropiate to import the sui-theme where their tokens are being used and that's inside the `settings` file.

Reply:
but in those components where it is used, its not a must that its removed from there and/or only included in the settings file, right?

Reply:
But here was used only because the settings where using them.

If I put the theme here and remove it from the settings, it would happen what you're mentioning in the first comment. You won't know where the tokens from the settings come from.

I'm going to add again the line here to keep importing the @s-ui/theme as it was before but that line would do nothing.

Reply:
I've added the line importing again the @s-ui/theme in the `index.scss`: https://github.com/SUI-Components/sui-components/pull/1946/commits/8f84f42aba6c8827b0d283a522ead529062d9ec3